package.json

```diff
@@ -24,7 +24,7 @@
   "scripts": {
     "build": "node ./node_modules/gulp/bin/gulp.js",
     "ensure-images": "node ./node_modules/gulp/bin/gulp.js ensure-images-optimized",
-    "serve": "yarn run ensure-images && node ./node_modules/polyserve/bin/polyserve -p 5101 -P h2 --proxy-path api --proxy-target http://localhost:9000/api/"
+    "serve": "yarn run ensure-images && node ./node_modules/polyserve/bin/polyserve -p 5101 --proxy-path api --proxy-target http://localhost:9000/api/"
```

Comment:
Non-https should be on port 5100 :)

Reply:
We run this on port 5101 because the front-end runs already on port 5100. Otherwise we cannot run them next to each other.